UofS-Pulse-Binfo/tripal_germplasm_importer
# New germplasm accession importer

ReadtheDocs has been updated in module, but it's not updated on [VCF Filter: Documentation](https://vcf-filter.readthedocs.io/en/latest/) yet since its not merged to master.
ReadtheDocs can be generated locally by following [KP guide](https://knowpulse.readthedocs.io/en/latest/developer_docs/rtd.html).
Please follow ReadtheDocs, and also run a job after installation to make sure ontology files can be imported.


One example test file for accession importer is provided in module in `tripal_germplam_importer/tests/test_files/`.

How to test the module:
1. need to re-install this module
    1.a disable this module and then uninstall it
    1.b update module and change to branch 12-new_class_GermplasmAccessionImporter
    1.c enable module(run a job if you are on dev), and go to Admin->People->Permissions to give permissions to "Importer: Use the Germplasm Cross Importer" and "Importer: Use the Germplasm Accession Importer"
2. now both loaders should appear in Admin->Data Loaders, it can be tested with an example file included in the moduel in tests/test_files
3. ReadtheDocs is updated, but it required to run `make html` under directory `docs/` on terminal then open docs/_build_html/index.html
